index.html

```diff
@@ -47,6 +48,160 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
     </ul>
   </section>
 
+<!-- works section -->
+
+  <section class="container">
+    <section id="works-section">
+      <h1 class="working">My Recent <br>Works</h1>
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+    <div class="grid-section">
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
```

Comment:
[Optional]

- [ ] Some elements are overflowing and the page is not completely centered, I suggest fixing these issues for a better looking website 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205452927-8b56baab-c479-4e02-af25-f6aa80126748.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205452951-9a58c339-3c74-4b09-9b31-f7b5922de79c.png)